### PR TITLE
Update citation.cff for v1.2

### PR DESCRIPTION
**Pull request by registerrier**
**Description**

This pull request updates citation.cff for the 1.2 release. Please fill your correct informations if you have contributed and wish to appear as author.

**Dear reviewer**

### CODE REVIEW

**Comment by bkhelifi** [comment]
I give you here some info:
- given-names: Paula
  family-names: Kornecki
  affiliation: Laboratoire Univers et Théories, Observatoire de Paris, Université PSL, Université Paris Cité, CNRS, F-92190 Meudon, France
  orcid: https://orcid.org/0000-0002-2706-7438
  

- given-names: Daniel
  family-names: Morcuende
  affiliation: Institute of Particle and Cosmos Physics, Instituto de Astrofísica de Andalucía, Universidad Complutense de Madrid
  orcid: https://orcid.org/0000-0001-9400-0922

File: CITATION.cff [hunk 1]
```diff
@@ -51,10 +63,18 @@ authors:
   affiliation: "Université Paris Cité, CNRS, Astroparticule et Cosmologie, F-75013\
     \ Paris, France"
   orcid: "https://orcid.org/0000-0001-6876-5577"
+- given-names: Paula
+  family-names: Kornecki
+  affiliation:
+  orcid:
```
Comment (bkhelifi) [inline CITATION.cff:69]:
```suggestion
- given-names: Paula
  family-names: Kornecki
  affiliation: Laboratoire Univers et Théories, Observatoire de Paris, Université PSL, Université Paris Cité, CNRS, F-92190 Meudon, France
  orcid: https://orcid.org/0000-0002-2706-7438
```

File: CITATION.cff [hunk 2]
```diff
@@ -63,6 +83,10 @@ authors:
   family-names: Mohrmann
   affiliation: "Max Planck Institute for Nuclear Physics, Heidelberg, Germany"
   orcid: "https://orcid.org/0000-0002-9667-8654"
+- given-names: Daniel
+  family-names: Morcuende
+  affiliation:
+  orcid:
```
Comment (bkhelifi) [inline CITATION.cff:89]:
```suggestion
- given-names: Daniel
  family-names: Morcuende
  affiliation: Institute of Particle and Cosmos Physics, Instituto de Astrofísica de Andalucía, Universidad Complutense de Madrid
  orcid: https://orcid.org/0000-0001-9400-0922
```

**Comment by registerrier** [comment]
Hi @StFroese , @nbiederbeck , @JBernete , can you check and complete your informations for the citation.cff (author list) for v1.2?

File: CITATION.cff [hunk 3]
```diff
@@ -24,11 +24,19 @@ identifiers:
...
+   \ Gif-sur-Yvette, France"
+  orcid: "https://orcid.org/0000-0002-6606-2816"
+- given-names: Juan
+  family-names: Bernete
+  affiliation:
+  orcid:
+- given-names: Noah
+  family-names: Biederbeck
+  affiliation: "Astroparticle Physics Group, TU Dortmund University, Germany"
+  orcid:
```
Comment (nbiederbeck) [inline CITATION.cff:39]:
```suggestion
  orcid: "https://orcid.org/0000-0003-3708-9785"
```

File: CITATION.cff [hunk 4]
```diff
@@ -38,36 +46,42 @@ authors:
...
-  affiliation: "Université Paris-Saclay, CNRS/IN2P3, IJCLab, 91405 Orsay, France"
-- given-names: Luca
-  family-names: Giunti
-  affiliation: "Université Paris Cité, CNRS, Astroparticule et Cosmologie, F-75013\
-    \ Paris, France"
-  orcid: "https://orcid.org/0000-0002-3395-3647"
+- given-names: Stefan
+  family-names: Fröse
+  affiliation: "Astroparticle Physics Group, TU Dortmund University, Germany"
+  orcid:
```
Comment (StFroese) [inline CITATION.cff:52]: `orcid: 0000-0003-1832-4129`

Comment (StFroese) [inline CITATION.cff:52]:
```suggestion
  orcid: https://orcid.org/0000-0003-1832-4129
```

File: CITATION.cff [hunk 5]
```diff
@@ -24,11 +24,19 @@ identifiers:
...
-    \ del Cosmos (ICCUB), Universitat de Barcelona (IEEC-UB), Spain"
-  orcid: "https://orcid.org/0000-0001-8816-4920"
+- given-names: Fabio
+  family-names: Acero
+  affiliation: "Université Paris-Saclay, Université Paris Cité, CEA, CNRS, AIM, F-91191\
+   \ Gif-sur-Yvette, France"
+  orcid: "https://orcid.org/0000-0002-6606-2816"
+- given-names: Juan
+  family-names: Bernete
+  affiliation:
```
Comment (JBernete) [inline CITATION.cff:34]:
```suggestion
  affiliation: "Centro de Investigaciones Energéticas Medioambientales y Tecnológicas (CIEMAT), Madrid, Spain"
```

File: CITATION.cff [hunk 6]
```diff
@@ -24,11 +24,19 @@ identifiers:
...
+  orcid:
```
Comment (JBernete) [inline CITATION.cff:35]:
```suggestion
  orcid: "https://orcid.org/0000-0002-8108-7552"
```

File: CITATION.cff [hunk 7]
```diff
@@ -85,11 +104,20 @@ authors:
...
 - given-names: Atreyee
   family-names: Sinha
   affiliation: "EMFTEL department and IPARCOS, Universidad Complutense de Madrid,\
     \ 28040 Madrid, Spain"
   orcid: "https://orcid.org/0000-0002-9238-7163"
+- given-names: Hanna
+  family-names: Stapel
+  affiliation: "Université Paris Cité, CNRS, Astroparticule et Cosmologie, F-75013\
+    \ Paris, France"
+  orcid:
```
Comment (bkhelifi) [inline CITATION.cff:120]: If Hanna has no orcid, this line should be removed

**Comment by bkhelifi** [comment]
There is a typo in: 

identifiers:
- description: "Zenodo repository of Gammapy"
    type: doi
    value: 10.5281/zenodo.470148

to

identifiers:
  - description: "Zenodo repository of Gammapy"
    type: doi
    value: 10.5281/zenodo.470148

**Comment by bkhelifi** [comment]
Also here:
```
contact:
- email: GAMMAPY-COORDINATION-L@IN2P3.FR
  name: "Coordination committee of the Gammapy project"

```
to 

```
contact:
  - email: GAMMAPY-COORDINATION-L@IN2P3.FR
    name: "Coordination committee of the Gammapy project"
```

File: CITATION.cff (hunk 7 above)
Comment (registerrier) [inline CITATION.cff:120]:
```suggestion
```

**Comment by registerrier** [comment]
Hi @neutrinoceros @kurtmckee, we are preparing release Gammapy v1.2 release. Do you want to appear as author? If yes please modify/fill the corresponding entries in the citation.cff for this release.
Thanks!

**Comment by neutrinoceros** [comment]
@registerrier thanks for the ping, but I don't mind not being listed ! Happy release !

**Comment by kurtmckee** [comment]
@registerrier I'm assuming I can open a PR to modify `citation.cff` against this branch. Is that acceptable?

**Comment by registerrier** [comment]
> @registerrier I'm assuming I can open a PR to modify `citation.cff` against this branch. Is that acceptable?

Yes sure. Or via comments.

**Comment by kurtmckee** [comment]
I've submitted registerrier/gammapy#13 to update my info. Thanks for the ping!

**Review by kurtmckee — APPROVED** [review]

File: CITATION.cff [hunk 8]
```diff
@@ -85,11 +104,19 @@ authors:
   family-names: Remy
   affiliation: "Max Planck Institute for Nuclear Physics, Heidelberg, Germany"
   orcid: "https://orcid.org/0000-0002-8815-6530"
+- given-names: Clément
+  family-names: Robert
+  affiliation:
+  orcid:
```
Comment (registerrier) [inline CITATION.cff:110]:
```suggestion
```

File: CITATION.cff [hunk 9]
```diff
@@ -129,7 +129,8 @@ authors:
     \ 2, 91058 Erlangen, Germany"
   orcid: "https://orcid.org/0000-0002-7378-4024"
 preferred-citation:
-  authors: ["The Gammapy team"]
+  authors:
+  - family-names: "the Gammapy team"
```
Comment (kurtmckee) [inline CITATION.cff:133]: I think that "The Gammapy team" reads more easily.

```suggestion
  - family-names: "The Gammapy team"
```

File: codemeta.json [hunk 10]
```diff
@@ -208,9 +268,8 @@
         "regions>=0.5.0",
         "pyyaml>=5.3",
         "click>=7.0",
-        "pydantic>=2.5",
+        "pydantic>=2.5.0",
         "iminuit>=2.8.0",
         "matplotlib>=3.4"
-    ],
-    "referencePublication": "https://doi.org/10.1051/0004-6361/202346488"
+    ]
```
Comment (bkhelifi) [inline codemeta.json:274]:
```suggestion
    ],
    "programmingLanguage": "Python",
    "referencePublication": "https://doi.org/10.1051/0004-6361/202346488"    
```

File: codemeta.json [hunk 11]
```diff
@@ -184,7 +244,7 @@
     "license": "https://spdx.org/licenses/BSD-3-Clause",
     "name": "Gammapy: Python toolbox for gamma-ray astronomy",
     "url": "https://gammapy.org/",
-    "softwareVersion": "v1.1",
+    "softwareVersion": "v1.2",
```
Comment (bkhelifi) [inline codemeta.json:247]:
```suggestion
    "version": "v1.2",
```

**Comment by StFroese** [comment]
I think we are missing @JanaKonrad, she worked on this PR with me #4740

**Comment by registerrier** [comment]
> I think we are missing @JanaKonrad, she worked on this PR with me #4740

Thanks, @StFroese . That's correct. The code that reads contribution needs to better check for commit co-authors.
Apparently, we also miss @jdjuvsla who did the review.

File: codemeta.json (hunk 10 above)
Comment (registerrier) [reply codemeta.json:274]: This will require changes in `dev/codemeta.py` as the file automatically generated.

**Comment by JanaKonrad** [comment]
> I think we are missing @JanaKonrad, she worked on this PR with me #4740 

Hi, thanks :) This is my personal data:

given-names: Jana
family-names: Konrad
affiliation: "Astroparticle Physics Group, TU Dortmund University, Germany"
orcid: "https://orcid.org/0009-0000-1257-4771"

**Comment by jdjuvsla** [comment]
Hi, 

for me this would be:

given-names: Julia
family-names: Djuvsland
affiliation: "University of Bergen, Norway and Max Planck Institute for Nuclear Physics, Heidelberg, Germany"
orcid: "https://orcid.org/0000-0002-6488-8219"

**Pull request merged**